core/xml.js

```diff
@@ -10,63 +10,69 @@
  */
 'use strict';
 
-/**
- * @name Blockly.Xml
- * @namespace
- */
-goog.provide('Blockly.Xml');
-goog.require('Blockly.Events');
-goog.require('Blockly.inputTypes');
-goog.require('Blockly.utils');
-goog.require('Blockly.utils.dom');
-goog.require('Blockly.utils.Size');
-goog.require('Blockly.utils.xml');
-
-goog.requireType('Blockly.Block');
+goog.module('Blockly.Xml');
+goog.module.declareLegacyNamespace();
+
+/* eslint-disable-next-line no-unused-vars */
+const Block = goog.requireType('Blockly.Block');
+const Events = goog.require('Blockly.Events');
+/* eslint-disable-next-line no-unused-vars */
+const Connection = goog.requireType('Blockly.Connection');
+/* eslint-disable-next-line no-unused-vars */
+const Field = goog.requireType('Blockly.Field');
+const Size = goog.require('Blockly.utils.Size');
+/* eslint-disable-next-line no-unused-vars */
+const VariableModel = goog.requireType('Blockly.VariableModel');
+/* eslint-disable-next-line no-unused-vars */
+const Workspace = goog.requireType('Blockly.Workspace');
+const dom = goog.require('Blockly.utils.dom');
+const inputTypes = goog.require('Blockly.inputTypes');
+const xml = goog.require('Blockly.utils.xml');
```

Comment:
`Xml` and `xml` in the same file feels like a recipe for confusion. I suggest `utilsXml` or xmlUtils`. @gonfunko had the same thing in one of his recent PRs.